distancematrix.go

```diff
@@ -127,6 +133,9 @@ type DistanceMatrixRequest struct {
 	// ArrivalTime specifies the desired time of arrival for transit requests, in seconds since midnight, January 1, 1970 UTC. You cannot
 	// specify both `DepartureTime` and `ArrivalTime`. Optional.
 	ArrivalTime string
+	// TrafficModel determines the type of model that will be used when determining travel time when using depature times in the future
+	// options are TrafficModelBestGuess, TrafficModelOptimistic or TrafficModelPessimistic. Optional. Default is TrafficModelBestGuess
+	TrafficModel TrafficModel
```

Comment:
That embeds the value, which is subtly different. This is fine - `DistanceMatrixRequest` doesn't really _inherit_ from `TrafficModel`.